templates/dynamodb.yml

```diff
@@ -0,0 +1,42 @@
+Resources:
+  UserTable:
```

Comment:
This is pretty cool. I didn't know we could break them out like this into multiple files!